### PR TITLE
support and check ERC165

### PR DESCRIPTION
**Pull request by 0x0Louis**

### CODE REVIEW

File: src/LBToken.sol [hunk 1]
```diff
@@ -328,4 +342,22 @@ contract LBToken is ILBToken {
...
+    function _verifyLBTokenSupport(address _target) private returns (uint256 supported) {
+        bytes4 selectorERC165 = IERC165.supportsInterface.selector;
+        bytes4 ILBTokenInterfaceId = type(ILBToken).interfaceId;
+
+        if (_target.code.length > 0)
+            assembly {
+                mstore(0x00, selectorERC165)
+                mstore(0x04, ILBTokenInterfaceId)
+
+                supported := staticcall(30000, _target, 0x00, 0x24, 0x00, 0x20)
```
Comment (Mathieu-Be) [inline src/LBToken.sol:358]: How did you get the gas value to send to the call ?

File: src/LBToken.sol [hunk 2]
```diff
@@ -328,4 +342,22 @@ contract LBToken is ILBToken {
...
+                supported := or(supported, mload(0x00))
```
Comment (Mathieu-Be) [inline src/LBToken.sol:359]: Is there any particular reason to not do `and` here and then checking `supported == true/false` ? Just out of curiosity, I think that's how I would have done it.

Comment (0x0Louis) [reply src/LBToken.sol:359]: Yes, as `3 & 5 = 1`, but I think it would be easier to read if we check that both results are exactly 1

Comment (0x0Louis) [reply src/LBToken.sol:359]: Oh but `1 | 0 = 1`, I'll check for exact outputs

**Review by Mathieu-Be — APPROVED** [review]

**Review by Mathieu-Be — APPROVED** [review]

**Pull request merged**